src/docfx/build/context/Input.cs

```diff
@@ -127,7 +127,7 @@ public FilePath[] ListFilesRecursive(FileOrigin origin, string dependencyName =
                         .ToArray();
 
                 case FileOrigin.Dependency:
-                    var dependencyPath = _restoreMap.GetGitRestorePath(_config.Dependencies[dependencyName], _docsetPath);
```

Comment:
it may be not always relative to current root docset

Reply:
like fallback repo has it's own template relative path, and loc is using overwrite config